docs/installation.rst

```diff
@@ -0,0 +1,103 @@
+Installation
+============
+
+The page has information for installing ``openfe``, installing software
+packages that integrate with ``openfe``, and testing that your ``openfe``
+installation is working.
+
+``openfe`` currently only works on POSIX system (macOS and UNIX/Linux). It
+is tested against Python 3.9 and 3.10.
+
+Installing ``openfe``
+---------------------
+
+When you install ``openfe`` through any of the methods described below, you
+will install both the core library and the command line interface.
```

Comment:
I'd include that the CLI is the prefix `openfe` (or is it `openfecli`?)

Reply:
Clarify: do you mean: 

* the command, called from your shell, is `openfe`?
* or, the import name for things within the CLI package is `openfecli`?

Both of these statements are true. I tend to think that making a big deal our of how to import the CLI will only lead to confusion (audience here is pretty inexperienced). Importing `openfecli` should almost never be done; exception is external plugins (which we haven't enabled yet and would need to be documented elsewhere if we enabled).

Reply:
yeah the shell version, I couldn't remember what it was called